addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -978,7 +978,7 @@ export class WysiwygAdapterComponent extends Wysiwyg {
             }
             const modelName = await this.websiteService.getUserModelName(resModel);
             const recordNameEl = imageEl.closest("body").querySelector(`[data-oe-model="${resModel}"][data-oe-id="${resID}"][data-oe-field="name"]`);
-            const recordName = recordNameEl ? `'${recordNameEl.textContent}'` : resID;
+            const recordName = recordNameEl ? `'${recordNameEl.textContent.replaceAll("/", "")}'` : resID;
```

Comment:
Is there other characters that might result in the same bug? Aren't we better off using the URL generated by the record rather than its name? (Since I assume it would have a clean path)

Reply:
There is no issue for `?`, `#`, `=`, `:`, `+`, `"`, `'` nor `%`: those are handled by the normal URL escaping.

Locating the record URL is different depending on the screen (e.g. list of blog posts vs single blog post) , the name is always within the edited cover.